src/Illuminate/Foundation/Console/Kernel.php

```diff
@@ -294,6 +299,33 @@ protected function bootstrappers()
     }
 
     /**
+     * Set environment if 'env' option if provided.
+     *
+     * @param  InputInterface  $input
+     * @return void
+     */
+    protected function setEnvironment($input)
+    {
+        if (! app()->runningInConsole()) {
```

Comment:
Yes indeed they are the same. If i use `$this->app` i can't load a different `.env` file, where as through app global helper i can.